sydestyle.sty

```diff
@@ -122,4 +122,9 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % End Title Page Stuff
 
+\newcommand*{\makesydebib}{
```

Comment:
Would be better to take a parameter and pass it into `\bibliography` to support multiple .bib files.
For example you can do `\bibliography{file1,references}`